src/Components/ProvisioningWizard/index.js

```diff
@@ -20,6 +21,10 @@ const ProvisioningWizard = ({ isOpen, onClose, image, ...props }) => {
   const [isConfirming, setConfirming] = React.useState(false);
   const [successfulLaunch, setLaunchSuccess] = React.useState();
 
+  React.useEffect(() => {
+    axios.defaults.headers.common['Correlation-Id'] = uuidv4();
```

Comment:
The correlation ID starts to be bit overused, I'd go with something more specific here, smthing like:
```suggestion
    axios.defaults.headers.common['Provisioning-Wizard-Id'] = uuidv4();
```
@lzap opinions?

Reply:
I don’t care which one to use, I already filed correlation-id for responses which is actually a trace id in our codebase. You know what, that is really weird, let me rename my header to Trace-Id and I propose this to be just Wizard-Id or maybe Ui-Id?

Anyways, you send a header but it will be thrown away by our backend. What is the purpose of this?

Reply:
I guess this is a UI part of a change, this is supposed to be followed by backend PR if I get that correctly?
Would you mind covering this in tickets @adiabramovitch, so we have more clarity on it?